### PR TITLE
New /download/iot/raspberry-pi-2-3 page with ubuntu server

### LINKED ISSUE #4955: [iot] update /download/iot/raspberry-pi-2-3 with server and move core page
[copy doc](https://docs.google.com/document/d/1KgnSBa8mbR7qwlxxtqbE5a1wXh5GjtVums8TIB7IGVw/edit#)

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

- Server version of raspberry pi page

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [/download/iot/raspberry-pi-2-3](http://0.0.0.0:8001/download/iot/raspberry-pi-2-3)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Compare to the [copy doc](https://docs.google.com/document/d/1KgnSBa8mbR7qwlxxtqbE5a1wXh5GjtVums8TIB7IGVw/edit#)

## Issue / Card

Fixes #4955

### CODE REVIEW

**Comment by pmahnke** [comment]
@caldav can you take a look?

**Comment by anasereijo** [comment]
@pmahnke 

I think we should start the page with a white background (because the secondary nav is also light grey) and then alternate with light grey. Other suggestions below:

chrome-extension___nlipoenfbbikpbjkfpfillcgkoblgpmj_edit html

File: templates/download/iot/raspberry-pi-2-3.html [hunk 1]
```diff
@@ -40,28 +43,27 @@ <h3>Minimum requirements</h3>
...
-              <li>You can verify the integrity of the files using the <a class="p-link--external" href="http://cdimage.ubuntu.com/ubuntu-core/18/stable/current/SHA256SUMS">SHA256SUM</a> and <a class="p-link--external" href="http://cdimage.ubuntu.com/ubuntu-core/18/stable/current/SHA256SUMS.gpg">SHA256SUM.gpg</a> files.
+              <li><a class="p-link--external" href="http://cdimage.ubuntu.com/releases/bionic/release/ubuntu-{{ lts_release_with_point }}-preinstalled-server-arm64+raspi3.img.xz">Ubuntu Server image for Raspberry Pi 3</a></li>
+              <li>You can verify the integrity of the files using the <a class="p-link--external" href="http://cdimage.ubuntu.com/releases/bionic/release/SHA256SUMS">SHA256SUM</a> and <a class="p-link--external" href="http://cdimage.ubuntu.com/releases/bionic/release/SHA256SUMS.gpg">SHA256SUM.gpg</a> files.
               </li>
             </ul>
           </div>
         </li>
-        {% include "./_flash-image.html" with card="microSD card" number=3 %}
+        {% include "./_flash-image.html" with card="microSD card" number=2 %}
```
Comment (caldav) [inline templates/download/iot/raspberry-pi-2-3.html]: Here we need to update the include text from "Copy the Ubuntu Core image on the microSD" to "Copy the Ubuntu image on the microSD"

File: templates/download/iot/raspberry-pi-2-3.html [hunk 2]
```diff
@@ -70,14 +72,47 @@ <h3 class="p-list-step__title">
             </ol>
           </div>
         </li>
-        {% include "./_first-boot-setup.html" with number=5 %}
-        {% include "./_login.html" with number=6 %}
+        {% include "./_login.html" with number=4 %}
```
Comment (caldav) [inline templates/download/iot/raspberry-pi-2-3.html]: This login step doesn't apply. 

Copy to use: "When prompted to log in, use "ubuntu" for the username and the password. You will be asked to change this default password after you log in."

File: templates/download/iot/raspberry-pi-2-3.html [hunk 3]
```diff
@@ -70,14 +72,47 @@ <h3 class="p-list-step__title">
...
+            </li>
+            <li>
+              <div class="p-code-snippet">
+                <input class="p-code-snippet__input" value="sudo apt-get install kubuntu-desktop" readonly="readonly">
+                <button class="p-code-snippet__action">Copy to clipboard</button>
+              </div>
+            </li>
+          </ul>
+        </li>
+        <li>For more details about Raspberry Pi specific packages included with this image and further customisations, such as accelerated video drivers and optional PPAs, you can refer to the <a class="p-link--external" href="https://wiki.ubuntu.com/ARM/RaspberryPi#Packages">RaspberryPi wiki</a>.</li>
```
Comment (caldav) [inline templates/download/iot/raspberry-pi-2-3.html]: optional PPAs -> optional package repositories

File: templates/download/iot/raspberry-pi-2-3.html (hunk 1 above)
Comment (pmahnke) [reply templates/download/iot/raspberry-pi-2-3.html]: I removed core from every mention

File: templates/download/iot/installation-media.html [hunk 4]
```diff
@@ -146,7 +146,7 @@ <h2>On Windows</h2>
             <li class="p-list-step__item col-8">
                 <p class="p-list-step__title">
```
Comment (deadlight) [inline templates/download/iot/installation-media.html]: Not introduced with your changes but this indentation should be fixed too

**Comment by pmahnke** [comment]
@deadlight fixed indentation

**Pull request merged**